src/sdk/namespace_fs.js

```diff
@@ -1426,6 +1426,15 @@ class NamespaceFS {
         }
     }
 
+    _is_force_md5_enabled(object_sdk) {
+        let md5_enabled = config.NSFS_CALCULATE_MD5;
+        // If exists specific Bucket/Account settings should override global settings
+        if (!_.isUndefined(this.force_md5_etag) || !_.isUndefined(object_sdk?.requesting_account?.force_md5_etag)) {
+            md5_enabled = (this.force_md5_etag || object_sdk?.requesting_account?.force_md5_etag);
```

Comment:
If somehow the `NSFS_CALCULATE_MD5` in the account was set to `true` and the bucket was set to `false` what do we expect the value to be?
I thought it should be determined by the bucket (if exists), then by account (if exists), and then by config (must exist).